codesign/test/file_codesign_visitor_test.dart

```diff
@@ -591,6 +591,20 @@ void main() {
     });
 
     test('visitBinary codesigns binary with / without entitlement', () async {
+      codesignVisitor = cs.FileCodesignVisitor(
+        codesignCertName: randomString,
+        codesignUserName: randomString,
+        appSpecificPassword: randomString,
+        codesignAppstoreId: randomString,
+        codesignTeamId: randomString,
+        googleCloudStorage: googleCloudStorage,
+        fileSystem: fileSystem,
+        processManager: processManager,
+        rootDirectory: rootDirectory,
+        filePaths: [randomString],
+        dryrun: false,
+        notarizationTimerDuration: const Duration(seconds: 0),
```

Comment:
```suggestion
        notarizationTimerDuration: Duration.zero,
```

Reply:
Neat, this is very helpful! Thanks for pointing this out